internal/install/execution/install_status.go

```diff
@@ -9,32 +9,32 @@ import (
 	"github.com/newrelic/newrelic-cli/internal/utils"
 )
 
-type StatusRollup struct {
+type InstallStatus struct {
 	Complete        bool `json:"complete"`
 	DocumentID      string
-	EntityGUIDs     []string `json:"entityGuids"`
-	Statuses        []Status `json:"recipes"`
-	Timestamp       int64    `json:"timestamp"`
-	LogFilePath     string   `json:"logFilePath"`
-	statusReporters []StatusReporter
+	EntityGUIDs     []string       `json:"entityGuids"`
+	Statuses        []RecipeStatus `json:"recipes"`
+	Timestamp       int64          `json:"timestamp"`
+	LogFilePath     string         `json:"logFilePath"`
+	statusReporters []StatusSubscriber
```

Comment:
Consider rename `statusReporters` to `statusSubscribers` to match the reference type.

Reply:
👍